slang-com-helper.h

```diff
@@ -0,0 +1,107 @@
+﻿#ifndef SLANG_COM_HELPER_H
```

Comment:
I'm still leaning toward something like `slang-utils.h` just so that we don't expose the notion of Slang being a "COM" API in terms of our naming and everything, but this is fine for now.